03-single-hop-ipv6-icmp/03-single-hop-ipv6-icmp.md

```diff
@@ -44,3 +44,21 @@ ICMPv6 echo request/reply exchange between two native nodes.
 ### Result
 
 <1% packets lost on the pinging node.
+
+Task #10 - ICMPv6 stress test on native (1 hour)
```

Comment:
oups, I merged this a bit fast, this should be `task #04`.

Reply:
You are right! However, I need to leave now. Can you fix this?